core/config/project_settings.cpp

```diff
@@ -1485,6 +1485,10 @@ ProjectSettings::ProjectSettings() {
 	GLOBAL_DEF("display/window/subwindows/embed_subwindows", true);
 	// Keep the enum values in sync with the `DisplayServer::VSyncMode` enum.
 	custom_prop_info["display/window/vsync/vsync_mode"] = PropertyInfo(Variant::INT, "display/window/vsync/vsync_mode", PROPERTY_HINT_ENUM, "Disabled,Enabled,Adaptive,Mailbox");
+
+	GLOBAL_DEF("display/window/frame_pacing/android/enable_frame_pacing", true);
+	GLOBAL_DEF(PropertyInfo(Variant::INT, "display/window/frame_pacing/android/swappy_mode", PROPERTY_HINT_ENUM, "pipeline_forced_on,auto_fps_pipeline_forced_on,auto_fps_auto_pipeline"), 2);
```

Comment:
Follow-up comment to place this under a `#if defined(SWAPPY_FRAME_PACING_ENABLED)` guard so the settings are only available when swappy is compiled into the build.

Reply:
Done.

Reply:
CI is unhappy with this because it sees that enable_frame_pacing should not exist and tags the problem with ProjectSettings.xml

See https://github.com/godotengine/godot/actions/runs/11428066052/job/31792909945

Reply:
Project settings should be exposed regardless of the build setting to avoid exactly this error. But setting the project setting should be a no-op, or at most print a warning that the engine is compiled without support for that feature